servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/HeaderUtils.java

```diff
@@ -119,9 +123,9 @@ private static void updateRequestContentLengthNonZero(final int contentLength, f
         }
     }
 
-    private static void updateRequestContentLength(final int contentLength, final HttpHeaders headers) {
```

Comment:
Surprised to see that `contentLength` is not a `long` argument

Reply:
This code executes only when we aggregate payload body into a single `Buffer`. The `Buffer` has the maximum length of `int`: `readableBytes()` returns `int`. So, we should not care about int overflow here.